src/hack/azext_hack/_database_utils.py

```diff
@@ -118,24 +119,25 @@ def __get_cosmosdb_steps(self, cmd, name, location):
             DatabaseAccountCreateUpdateParameters,
             DatabaseAccountKind,
             Location,
-            ConsistencyPolicy
+            MongoDBDatabaseCreateUpdateParameters
         )
 
         server_params = {
             'resource_group_name': name,
             'account_name': name,
-            'params': DatabaseAccountCreateUpdateParameters(
+            'create_update_parameters': DatabaseAccountCreateUpdateParameters(
                 location=location,
                 locations=[Location(
                     location_name=location,
                     failover_priority=0,
                     is_zone_redundant=False)],
                 kind=DatabaseAccountKind.mongo_db.value,
-                consistency_policy=ConsistencyPolicy(
-                    default_consistency_level=1,
-                    max_staleness_prefix=100,
-                    max_interval_in_seconds=5
-                )
+                capabilities=[
+                    Capability(name='EnableMongo'),
+                    Capability(name='EnableServerless'),
+                    Capability(name='DisableRateLimitingResponses')
+                ],
```

Comment:
May I know why delete the consistency policy and disable the rate limiting response?

Reply:
Because it's the only thing that works. And also, there was no reason for the consistency policy being defined here in the first place. FWIW the server default is the same as what was set here explicitly.

When I set the consistency policy I got HTTP 400 errors with the serverless setting. The rate limiting response is the value set when you create a serverless Cosmos DB account with Mongo via the Azure poral. Not sure why this is necessary, but I am replicating the portal experience.

Ultimately the Cosmos DB SDK will change and support this directly, but there is a lot to gain for the users of `az hack` by using the serverless offering now and not having to use the portal to set this up.

It's a hack for `az hack` :)